README.md

```diff
@@ -3,9 +3,36 @@ UE4 Project which includes examples to use rclUE.
 ## Maps
 - PubSub: BP and C++ ROS2 publisher subscriber example.
 - Turltebot3 Benchmark: BP and C++ ROS2 turtleobot3 navigation. Burger is implemented in C++ and Waffle is implemented in BP
-## Setup
-1. clone this repo
-2. ./run_editor.sh
+
+## Setup for Ubuntu 20.04 & ROS2 foxy
+
+Retrieve the repository with all required submodules:
+```
+git clone --recurse-submodules git@github.com:rapyuta-robotics/turtlebot3-UE.git
+```
+Generate VSCode build configurations:
+```
+cd turtlebot3-UE
+./update_project_files.sh
+```
+Beware that the update_project_files script considers that Unreal Engine and turtlebot3-UE are located in the same folder. If that's not the case, make a copy of the script with your correct Unreal Engine folder.
+
+If the script fails because of obsolete temporary files, remove them:
+```
+rm -r Binaries Intermediate Saved Plugins/rclUE/Binaries Plugins/rclUE/Intermediate Plugins/RapyutaSimulationPlugins/Binaries Plugins/RapyutaSimulationPlugins/Intermediate
+```
+Build the plugins and UE4 project:
+```
+make turtlebot3Editor
```

Comment:
For sake of possible packaging, should we add `turtlebot3` to be `make`d also?

Reply:
it is better to write somewhere but make turtlebot3 take a bit long time to build.

Reply:
just did a clean install and the makefile is not present in the folder.
But the command line used by vs code, that can be called directly in the terminal, is :
`../UnrealEngine/Engine/Build/BatchFiles/Linux/Build.sh turtlebot3Editor Linux Development <path to turtlebot3-UE folder>/turtlebot3.uproject`

Reply:
There is also the update_project_files.sh script which will generate the makefile, and a build_editor.sh script that will build it, and a run_editor.sh script which will run it.

Reply:
The update_project_files.sh does not create the makefile, in my case. And I don't know how I manage to get a makefile in this folder before...

Reply:
Finally I could fix the makefile generation when callng update_project_files.sh :
Added these lines 
```
	<ProjectFileGenerator>
		<Format>Make</Format>
	</ProjectFileGenerator>
```
in the configuration part of the file `UnrealEngine/Engine/Saved/UnrealBuildTool/BuildConfiguration.xml`